pkg/client/clientset_generated/release_1_2/clientset.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package release_1_2
+
+import (
+	"github.com/golang/glog"
+	core_v1 "k8s.io/kubernetes/pkg/client/typed/generated/core/v1"
+	extensions_v1beta1 "k8s.io/kubernetes/pkg/client/typed/generated/extensions/v1beta1"
+	unversioned "k8s.io/kubernetes/pkg/client/unversioned"
```

Comment:
Is this package necessary?

I see you use it for the config. Can the config be split out of the unversioned client? @krousey